sonic-xcvrd/tests/test_xcvrd.py

```diff
@@ -42,6 +42,20 @@
 global_media_settings = media_settings_with_comma_dict['GLOBAL_MEDIA_SETTINGS'].pop('1-32')
 media_settings_with_comma_dict['GLOBAL_MEDIA_SETTINGS']['1-5,6,7-20,21-32'] = global_media_settings
 
+
```

Comment:
Let's separate out the testcases from xcvrd, you can create a new file test_y_cable.py, move these new tests and other 
ycable tests into that.